docs/pages/connect-your-client/vnet.mdx

```diff
@@ -13,12 +14,13 @@ Teleport. Underneath, VNet authenticates the connection with your credentials, u
 routing technology](../architecture/tls-routing.mdx) as `tsh proxy app`. This is all done
 client-side – VNet sets up a local DNS name server and a virtual network device.
 
-VNet is available in Teleport Connect and tsh on macOS, with Windows and Linux version in the works.
+VNet is available in Teleport Connect and `tsh` on macOS, with Windows and Linux version in the works.
```

Comment:
I would revert this. I don't think we want people running VNet directly via `tsh` any time soon.